inc/vite.php

```
@@ -29,17 +29,23 @@ private function load_manifest() {
         ];
 
         foreach ($paths as $path) {
-            if (file_exists($path)) {
-                $cache_key = 'djz_vite_manifest_v2_' . md5($path);
-                $cached    = get_transient($cache_key);
+            $cache_key = 'djz_vite_manifest_v2_' . md5($path);
```

PHPMD (2.15.0)

[error] The variable $cache_key is not named in camelCase. (undefined)

(CamelCaseVariableName)